spec/index.html

```diff
@@ -551,10 +551,10 @@ <h3>Overview</h3>
         is represented as a set of <a>quads</a> of the form `&lt; s, p, o, g >`
         where the graph component `g` is empty if and only if the
         <a data-lt="RDF triple">triple</a> `&lt; s, p, o >` is in the <a>default graph</a>.
-        This algorithm considers an RDF dataset to be a set of quads.
-        Two RDF datasets are considered to be isomorphic (i.e., the same modulo blank nodes),
-        if and only if they return the same canonically labeled list of quads
-        via <a>URDNA2015</a>.</p>
+        It is expected that, for two RDF datasets,
+        <a>URDNA2015</a> returns the same canonically labeled list of quads
+        if and only if the two datasets are isomorphic (i.e., the same modulo blank nodes).
```

Comment:
> `isomorphic (i.e., the same modulo blank nodes).`

This suggests that some blank node(s) could exist in either dataset and not in the other. This is not my understanding of isomorphism. Perhaps, `the same modulo blank node identifiers` or `the same modulo local identifiers of blank nodes` or something along these lines?

Reply:
I agree that this would be even better.

Reply:
Wonder if we amend a PR and merge it again... I'll try.

Reply:
```suggestion
        if and only if the two datasets are isomorphic (i.e., the same modulo blank node identifiers).
```

Reply:
"suggestions can not be applied while the pull request is closed" :-(

Reply:
made a new PR for that #85